fuel-core/src/model/fuel_block.rs

```diff
@@ -22,7 +22,7 @@ use std::{
     Into,
     From,
 )]
-pub struct BlockHeight(u32);
+pub struct BlockHeight(pub u32);
```

Comment:
we shouldn't need to expose this since we have all the derive_more::{From, Into} setup.